### PR TITLE
InitSkill Parser Added

### PR DESCRIPTION
**Pull request by Cr3ativeZen**

### CODE REVIEW

File: apps/gameserver/include/Skill.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <vector>
```
Comment (lafreak) [inline apps/gameserver/include/Skill.h:3]: Why is it needed?

File: apps/gameserver/include/Skill.h [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    std::int32_t LevelLimit = 0;
+    std::uint32_t Job = 0;
+    std::uint32_t RequiredSkillId = 0;
+    std::uint32_t RequiredSkillGrade = 0;
+    std::uint32_t MaxLevel = 0;
+    std::uint32_t MP = 0;
+    std::uint32_t LastTime = 0;
+    std::uint32_t CastTime = 0;
+    std::uint32_t CoolDown = 0;
+    std::uint32_t Unknown = 0; // third argument of delay
```
Comment (lafreak) [inline apps/gameserver/include/Skill.h:21]: Comment out until we are not sure what it is

File: apps/gameserver/src/Skill.cpp [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            break;
+        }
+        case A_MAXLEVEL:     MaxLevel     = param.pop(); break;
+        case A_MP:           MP           = param.pop(); break;
+        case A_LASTTIME:     LastTime     = param.pop(); break;
+        case A_DELAY:
+        {
+            CastTime = param.pop();
+            CoolDown = param.pop();
+            Unknown  = param.pop();
```
Comment (lafreak) [inline apps/gameserver/src/Skill.cpp:33]: Comment out

**Review by lafreak — APPROVED** [review]

File: apps/gameserver/include/Skill.h (hunk 1 above)
Comment (Cr3ativeZen) [reply apps/gameserver/include/Skill.h:3]: ups, done

File: apps/gameserver/include/Skill.h (hunk 2 above)
Comment (Cr3ativeZen) [reply apps/gameserver/include/Skill.h:21]: done

File: apps/gameserver/src/Skill.cpp (hunk 3 above)
Comment (Cr3ativeZen) [reply apps/gameserver/src/Skill.cpp:33]: done

**Review by lafreak — APPROVED** [review]

**Pull request merged**